packages/teams-js/src/internal/appUtil.ts

```diff
@@ -0,0 +1,162 @@
+import * as Handlers from '../internal/handlers'; // Conflict with some names
```

Comment:
The functions in these new util files I think would be better to leave into the original files (app.ts, dialog.ts, pages.ts), and we should just do whatever we need to to not export them for external customers. I totally get why you moved them to new utils files (they're shared between multiple files and putting them here makes it easy to not export them outside of the library), but files like tasks are eventually going to get deleted, at which point we wouldn't really want functionality to live in a utils file that was only called by one file (dialog.ts)

Reply:
functions under [app/pages/dialog]Utils are moved back to original files but abstract as helper functions still.